### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by johnie261**
### I and my coding partners did the following:

- Added a mobile menu
- Implemented DOM
- Styled the mobile menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -55,7 +57,7 @@ body {
   /* animation changes */
 }
 
-.headr .navbar a {
+.headr .navbar .links {
   font-style: normal;
   font-weight: 500;
   font-size: 15px;
```
Comment (Omar-Muhamad) [inline style.css:63]: - [ ] There is a side-scrolling in the mobile version and mobile menu to get to the hamburger icon and the close icon, So please fix that.

- [ ] Please make the background of the mobile nav menu blurry to match the given design.
[hint] You can do that using [backdrop-filter](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.php) CSS property.

chrome-capture-2023-0-31

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply style.css:69]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @johnie261 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**